app/javascript/vue/navigation/top_menu.vue

```diff
@@ -17,7 +17,7 @@
     <MenuDropdown
       class="ml-auto"
       v-if="this.settingsMenu && this.settingsMenu.length > 0"
-      :listItems="this.settingsMenu.map((item) => { return { text: item.name, url: item.url } })"
+      :listItems="this.settingsMenu.map((item) => { return { text: item.name, url: item.url } }).concat({ text: this.i18n.t('left_menu_bar.support_links.core_version'), modalTarget: '#aboutModal', url: '' })"
```

Comment:
this. is not needed here and this logic could go into a computed property ex. settingsMenuList